.helm/charts/guardian/values.yaml

```diff
@@ -0,0 +1,84 @@
+# Default values for guardian.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/dollarshaveclub/guardian
+  tag: "6a5389928765fe33aa37748d4cc633dc16f880e1"
+  pullPolicy: Always
+
+deploymentColor: ""
+ingressClass: "nginx"
+serviceAccountName: ''
+
+app:
+  port: 3000
+
+guardian:
+  logLevel: "warn"
+  address: "0.0.0.0"
+  redisAddress: "127.0.0.1:6379"
+  redisPoolSize: "1"
+  dogstatsdAddress: "127.0.0.1:8125"
+  reportOnly: "false"
+  limit: "100"
+  limitDuration: "1s"
+  limitEnabled: "true"
+  confUpdateInterval: 10s
+  synchronous: false
+  initConfig: true
```

Comment:
I would recommend setting the default value for `initConfig` to `false` and instead set this to true in the `dqa.yaml` value overrides file. 

Just to maintain the same behavior of the application defaulting this value to false. It would also provide some additional protection around unintentionally resetting or initializing redis is other environments outside of DQAs.

Reply:
Done. Also I changed the init function to only make changes if redis keys are missing.